### DISCUSSION TITLE
Allow dynamic values in "delay"?

### DISCUSSION THREAD

**Discussion by matheau** [excid3/noticed#101] [Q&A]
Is there a way to support dynamic values in "delay"? For example, instead of putting:

`deliver_by :email, mailer: 'CommentMailer', delay: 5.minutes, if: :email_notifications?`

… is there a way to do something like:

```
deliver_by :email, mailer: 'CommentMailer', delay: :dynamic_delay, if: :email_notifications?

def dynamic_delay
     5.minutes
end
```

**Comment by excid3** [comment]
Not currently, but I'd be open to a PR for that.
